test/integration/targets/collections_runtime_pythonpath/ansible-collection-python-dist-boo/ansible_collections/python/dist/plugins/modules/boo.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove the unnecessary boilerplate from this module. It doesn't need a copyright notice, nor does it need any classes. A simple main function, much like the ping module, is all that is needed. Doing that will cut this module down to about 25% of its current size and make it much more obvious what is going on in the test.

The same applies to the other test module.

Reply:
I wanted to avoid adding sanity ignore lines

Reply:
Sanity ignore entries will not be needed.

Module specific sanity tests such as validate-modules do not apply to test modules.

Reply:
Yep, I already see it. Something else failed then.